mk/tests.mk

```diff
@@ -111,14 +111,15 @@ $(foreach target,$(CFG_TARGET), \
 # for arm-linux-androidabi
 define DEF_ADB_DEVICE_STATUS
 CFG_ADB_DEVICE_STATUS=$(1)
+CFG_ANDROID_TARGET_TRIPLE=$(2)
```

Comment:
I think that this means that if you configure both `aarch64-linux-android` and `arm-linux-androideabi` that it won't work out so hot, but that's probably fine.